packages/sanity/src/desk/structureBuilder/Component.ts

```diff
@@ -1,114 +1,131 @@
 import {camelCase} from 'lodash'
-import {SerializeOptions, StructureNode, Serializable, Child} from './StructureNodes'
-import {SerializeError, HELP_URL} from './SerializeError'
-import {MenuItem, MenuItemBuilder, maybeSerializeMenuItem} from './MenuItem'
-import {MenuItemGroup, MenuItemGroupBuilder, maybeSerializeMenuItemGroup} from './MenuItemGroup'
+import {
+  StructureSerializeOptions,
+  StructureNode,
+  StructureSerializable,
+  StructureChild,
+} from './StructureNodes'
+import {SerializeError, STRUCTURE_HELP_URL} from './SerializeError'
+import {StructureMenuItem, StructureMenuItemBuilder, maybeSerializeMenuItem} from './MenuItem'
+import {
+  StructureMenuItemGroup,
+  StructureMenuItemGroupBuilder,
+  maybeSerializeMenuItemGroup,
+} from './MenuItemGroup'
 import {validateId} from './util/validateId'
-import {UserComponent} from './types'
-
-export interface Component extends StructureNode {
-  component: UserComponent
-  child?: Child
-  menuItems: MenuItem[]
-  menuItemGroups: MenuItemGroup[]
+import {StructureUserComponent} from './types'
+
+/** @beta */
+export interface StructureComponent extends StructureNode {
+  component: StructureUserComponent
+  child?: StructureChild
+  menuItems: StructureMenuItem[]
+  menuItemGroups: StructureMenuItemGroup[]
   options: {[key: string]: unknown}
 }
 
-export interface ComponentInput extends StructureNode {
-  component: UserComponent
-  child?: Child
+/** @beta */
+export interface StructureComponentInput extends StructureNode {
+  component: StructureUserComponent
+  child?: StructureChild
   options?: {[key: string]: unknown}
-  menuItems?: (MenuItem | MenuItemBuilder)[]
-  menuItemGroups?: (MenuItemGroup | MenuItemGroupBuilder)[]
+  menuItems?: (StructureMenuItem | StructureMenuItemBuilder)[]
```

Comment:
I guess this is the same kind of naming implications like values and options; probably used in userland code, so should get a callout in breaking docs?